src/layouts/Tutorial.tsx

```diff
@@ -34,33 +24,16 @@ import {
 import TableOfContents from "@/components/TableOfContents"
 import TooltipLink from "@/components/TooltipLink"
 import TutorialMetadata from "@/components/TutorialMetadata"
-import { Divider } from "@/components/ui/divider"
 import { mdxTableComponents } from "@/components/ui/Table"
 import YouTube from "@/components/YouTube"
 
 import { getEditPath } from "@/lib/utils/editPath"
 
-type ContentContainerProps = Pick<BoxProps, "children" | "dir">
-
-const ContentContainer = (props: ContentContainerProps) => {
-  const boxShadow = useToken("colors", "tableBoxShadow")
-
+const ContentContainer = (props: HTMLAttributes<HTMLDivElement>) => {
```

Comment:
Single use and not exported... Can we move `<MainArticle ...>` to the return statement?

Reply:
Definitely